container.nix

```diff
@@ -142,6 +142,14 @@ let
       description = "--user UID:...";
       property = "Group";
     };
+    
+    gidMap = quadletUtils.mkOption {
+      type = types.nullOr types.str;
```

Comment:
> This key can be listed multiple times.

https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html#gidmap.

Let's make it a list and have a plural name.

Reply:
Made the suggested changes. Their now lists and have plural names.